tests/system_metrics/test_system_metrics_logging.py

```diff
@@ -1,10 +1,21 @@
 import threading
 import time
 
+import pytest
+
 import mlflow
 from mlflow.system_metrics.system_metrics_monitor import SystemMetricsMonitor
 
 
+@pytest.fixture(scope="module", autouse=True)
+def clean_mlruns_dir():
```

Comment:
can we rename this fixture?

Reply:
should the scope be module? should we run this after each test?

Reply:
sorry my bad.